.github/workflows/bazel.yml

```diff
@@ -131,13 +153,56 @@ jobs:
         uses: rtCamp/action-slack-notify@12e36fc18b0689399306c2e0b3e0f2978b7f1ee7 # pin@v2.2.0
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_BAZEL_CI }}
-          SLACK_TITLE: "Bazel Build & Test Job `bazel build //...; bazel test //...`"
+          SLACK_TITLE: "Bazel Build & Test Job `bazel build //...; bazel test //...` ${{ matrix.bazel-config }}"
           SLACK_USERNAME: "Bazel Build & Test"
           SLACK_ICON_EMOJI: ":boom:"
           SLACK_COLOR: "#FF0000"
           SLACK_FOOTER: ' '
           MSG_MINIMAL: actions url,commit
 
+  if_bazel_build_and_test_success:
+    name: Run when bazel successful
+    needs: bazel_build_and_test
+    runs-on: ubuntu-latest
+    if: success() # Only run after all matrix jobs have passed
+    # See https://github.com/magma/magma/wiki/How-to-set-up-a-required-matrix-workflow-on-GitHub-actions
+    # or https://github.com/magma/magma/pull/13562 for more details.
+    # Store success output flag for report_result_bazel_build_and_test job
+    outputs:
+      success: ${{ steps.setoutput.outputs.success }}
+    steps:
+      - id: setoutput
+        run: echo "::set-output name=success::true"
+
+  report_result_bazel_build_and_test:
+    name: Bazel build and test status
+    runs-on: ubuntu-latest
+    if: always()
+    # This job always needs to run. It will be green if the bazel_build_and_test
+    # job was successful in all matrix jobs or if the job was skipped.
+    needs: [path_filter, bazel_build_and_test, if_bazel_build_and_test_success]
+    steps:
+      - run: |
+          bazel_build_and_test_success="${{ needs.if_bazel_build_and_test_success.outputs.success }}"
+          files_changed="${{ needs.path_filter.outputs.files_changed }}"
+
+          echo "The status of this job is determined by the statuses of the previous jobs in this workflow."
+          echo "For more details on this matrix workflow please look at the following wiki page or the PR #13562:"
+          echo "https://github.com/magma/magma/wiki/How-to-set-up-a-required-matrix-workflow-on-GitHub-actions"
+          echo "https://github.com/magma/magma/pull/13562"
+
+          echo "bazel_build_and_test_success: $bazel_build_and_test_success"
+          echo "files_changed: $files_changed"
+
+          if [[ $bazel_build_and_test_success == "true" || $files_changed != 'true' ]];
```

Comment:
There is a inconsistency here for `'` and `"` usage.

Reply:
Pushed